src/sbt-test/dependencies/integrationTest/etc/bom.xml

```diff
@@ -197,201 +197,5 @@
       <purl>pkg:maven/org.scala-lang/scala-reflect@2.12.20</purl>
       <modified>false</modified>
     </component>
-    <component type="library">
```

Comment:
why did these disappear? or does this just need a rebase?

Reply:
this pr contains the same change as in #95 introducing a .distinct to deduplicate components because the json format is not happy about duplicate entries. This change will probably disappear when I rebase.